salesforce-composite/2.7/modules/ROOT/pages/salesforce-composite-connector-examples.adoc

```diff
@@ -114,21 +115,21 @@ The following is an example of HTTP input for the *Transform message* component:
 
 === Add the Create SObject Tree Operation
 
-Add the *Create sobject tree* operation to connect with Salesforce and create an SObject tree:
+Add the *Create SObject Tree* operation to connect with Salesforce and create a SObject tree:
 
-. In the *Mule Palette* view, search for *Salesforce* and select the *Create sobject tree* operation.
-. Drag the *Create sobject tree* operation to the right of *Transform Message*.
-. Click the green plus sign (*+*) to the right of the *Connector configuration* field to access the Salesforce Composite global element configuration fields.
+. In the *Mule Palette* view, search for *Salesforce* and select the *Create SObject Tree* operation.
+. Drag the *Create SObject Tree* operation to the right of *Transform Message*.
+. Click the green plus sign (*+*) to the right of the *Connector Configuration* field to access the Salesforce Composite global element configuration fields.
 . Complete the fields using variables, as follows:
 .. In the *Create SObject tree* properties window, enter `Account` in the *SObject Root Type* field.
 .. In the properties file `src/main/mule/mule-app.properties`, declare and enter values for the variables in the global element.
 
 === Add the Second Transform Message Component
 
-This *Transform Message* component ... 
+This *Transform Message* component transforms the output result into a JSON:
 //What is this transform message component doing?
```

Comment:
If you answered the questions I asked in the comments, you can delete the comments